src/Marten/Events/Aggregation/FanOutEventDataOperator.cs

```diff
@@ -0,0 +1,44 @@
+using System;
+using System.Collections.Generic;
+using Marten.Events.Projections;
+
+namespace Marten.Events.Aggregation;
+
+internal abstract class FanOutOperator<TSource>: IFanOutRule
```

Comment:
@erdtsieck, I apologise for such partial reviews, but one last ask: could you rename this file back to `FanOutOperator`? I think this name is more explicit, and it helps to keep track of the file changes history.

Reply:
That's no problem. I understand. Resharper can be helpfull, but you have to watch all changes

Reply:
Thanks!